assembl/models/attachment.py

```diff
@@ -180,6 +183,29 @@ def __init__(self, *args, **kwargs):
                              name="anti_virus_status"),
                         server_default='unchecked')
 
+    def guess_extension(self):
+        extensions = guess_all_extensions(self.mime_type)
+        if extensions:
+            # somewhat random
+            return extensions[0]
+
+    def check_for_viruses(self, antivirus=None):
+        "Check if the file has viruses"
+        antivirus = antivirus or get_antivirus()
+        safe = antivirus.check(self.data, self.guess_extension())
```

Comment:
What is the reason to do work about file extension? Is it really valuable input for the antivirus? (does it not already do this work?)